packages/edit-post/src/components/layout/index.js

```diff
@@ -154,6 +146,7 @@ function Layout() {
 			<LocalAutosaveMonitor />
 			<EditPostKeyboardShortcuts />
 			<EditorKeyboardShortcutsRegister />
+			<SettingsSidebar />
```

Comment:
Why was this moved here?

Reply:
Our slot mechanism for complementary areas works with assumption multiple fills are rendered. Only the fill of the active complementary area is shown, but buttons to toggle the complementary area are rendered for all the areas.
Previously the core sidebar fill was only rendered when the sidebar was enabled, and we had a custom made toggle button for the sidebar.
This PR makes sure the core sidebars use the same logic as the plugin sidebars so we need to move SettingsSidebar to a position where it is unconditionoly rendered even if no sidebars are active. Otherwise, when no sidebars are active, we would not render the toggle button for core sidebars.